td/views.py

```diff
@@ -402,7 +401,7 @@ class AjaxLanguageListView(LanguageTableSourceView):
         "code",
         "iso_639_3",
         "name",
-        "direction",
+        "anglicized_name",
```

Comment:
@vleong2332 this is just for the table display not the json endpoint, it was a matter of just making room and i asked him to do this as I thought that was the least important thing to show on the website.  I had thought that this tables were just used for a quick reference / debugging.